starlite/testing/client/sync_client.py

```diff
@@ -666,11 +616,6 @@ def test_something(self, test_client: TestClient) -> None:
             return {}
         return self.get_session_data()
 
-    @staticmethod
-    def _create_session_cookies(backend: "CookieBackend", data: Dict[str, Any]) -> Dict[str, str]:
-        encoded_data = backend.dump_data(data=data)
-        return {cookie.key: cast("str", cookie.value) for cookie in backend._create_session_cookies(encoded_data)}
-
     async def _set_session_data_async(self, data: Dict[str, Any]) -> None:
```

Comment:
These should go into the `BaseClient`. In the async client, they can simply be exposed, and in the sync client, they can be wrapped as-is. Currently, there's a re-implementation of them in the async client, which is unnecessary.

Reply:
@provinzkraut Thank you, I noticed it while I was updating the docs and will fix it in the next commit.

Reply:
@provinzkraut  Could you check the changes made [here](https://github.com/starlite-api/starlite/pull/1045/commits/6c4ed0fe2602ae5d6c9ea41affeb26ffccaa8cc7)!